translations/combat.tex/de.po

```diff
@@ -227,6 +323,9 @@ msgid ""
 "\\end{center}\n"
 "\n"
 msgstr ""
+"\\begin{center}\n"
+"\\end{center}\n"
```

Comment:
```suggestion
```
Don't do those weird things. You can put "\n" in msgstr.

Reply:
This changes doesnt work. When i trigger workflows with only "\n" then i get following error in poedit and no "build-pdf-Check" of any language was succesful.  
![image](https://github.com/user-attachments/assets/1d2777f3-a40e-413e-900a-120202fd7f4d)

Is it necessary to change the "weird things". because the german pages look fine.

Reply:
In that case please do this:
```po
msgstr "\\empty"
"\n"
```